spatio_temporal_voxel_layer/src/spatio_temporal_voxel_grid.cpp

```diff
@@ -410,7 +432,7 @@ void SpatioTemporalVoxelGrid::ResetGridArea(
     const bool in_y_range = pose_world.y() > start.y && pose_world.y() < end.y;
     const bool in_range = in_x_range && in_y_range;
 
-    if(in_range == invert_area)
```

Comment:
I don't understand how this is a diff....

on the ros2 branch it's already != 

https://github.com/SteveMacenski/spatio_temporal_voxel_layer/blob/ros2/spatio_temporal_voxel_layer/src/spatio_temporal_voxel_grid.cpp#L413

Reply:
I'm guessing you just need to pull in main or rebase?

Reply:
Not near my computer right now but this points to the current code and if you look at the current code on the ros2 branch, it's already '!='

(It's late and most likely github is right and I'm wrong but I don't see it yet...)